mlflow/__init__.py

```diff
@@ -44,6 +46,14 @@
 
 _configure_mlflow_loggers(root_module_name=__name__)
 
+if sys.version_info.major == 2:
+    warnings.warn("Python 2 support for MLflow is deprecated and will be dropped in a future "
```

Comment:
Does this PR imply that we intend to preserve Python2 support in the next release of MLflow (1.7)? Otherwise, this message would be inaccurate for users on 1.7 (support will have already been dropped)

Reply:
The first version makes it sound like Python 2 has MLflow libraries. I think you mean:

MLflow support for Python 2 is deprecated and will be dropped in a future

Reply:
Thanks both, yep @dbczumar the idea is to preserve support for at least one version in 1.7 & drop it afterwards, per discussion with @mateiz, just to give users who may be stuck with Python 2 more advance notice